molecule/provisioner/ansible/plugins/libraries/molecule_vagrant.py

```diff
@@ -162,13 +162,9 @@
         virtualbox.cpus = provider['options']['cpus']
 
         if provider['options'] and provider['options']['linked_clone']
-          if Vagrant::VERSION =~ /^1.[89]/
-            virtualbox.linked_clone = provider['options']['linked_clone']
-          end
+          virtualbox.linked_clone = provider['options']['linked_clone'] if Gem::Version.new(Vagrant::VERSION) >= Gem::Version.new('1.8.0')
```

Comment:
While I understand this is a more ruby way of doing things with the one line setting of a conditional var, it will exceed the project limit line length.  Please use the Gem version comparison, but keep the conditional on a separate line.